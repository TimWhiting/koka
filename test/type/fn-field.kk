struct fn-field
  xrun: (string) -> io ()

fun main()
  val f = Fn-field(fn(s) println(s))
  f.xrun("Hello, World!")
