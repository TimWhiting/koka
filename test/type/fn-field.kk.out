type/fn-field/fn-field/call/xrun: (fn-field, string) -> io ()
type/fn-field/fn-field/get/xrun: (fn-field) -> ((string) -> io ())
type/fn-field/Fn-field: (xrun : (string) -> io ()) -> fn-field
type/fn-field/main: () -> io ()